rest/src/main/java/discord4j/rest/entity/RestGuild.java

```diff
@@ -264,9 +264,14 @@ public Mono<RoleData> createRole(RoleCreateRequest request, @Nullable String rea
         return restClient.getGuildService().createGuildRole(id, request, reason);
     }
 
-    public Flux<RoleData> modifyRolePositions(List<RolePositionModifyRequest> requests) {
+    public Flux<RoleData> modifyRolePositions(List<RolePositionModifyRequest> requests, @Nullable String reason) {
         return restClient.getGuildService().modifyGuildRolePositions(id,
-                requests.toArray(new RolePositionModifyRequest[0]));
+            requests.toArray(new RolePositionModifyRequest[0]), reason);
+    }
+
+    @Deprecated
```

Comment:
I don't think we should deprecate this method; there is no need to use the other rather than this one, especially if we don't want to add a reason. It's easier not to have to add a null argument.

Reply:
my idea for deprecate is most for remove a method not used i can make the null thing. but maybe can keep like currently.. not hurst anybode

Reply:
It's more than overrides like these are already present many times throughout the code and are better for developers.